tests/vnc_test.go

```diff
@@ -155,6 +161,54 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			_, err = wrappedRoundTripper.RoundTrip(req)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should connect to vnc with --proxy-only flag", func() {
+			By("Invoking virtctl vnc with --proxy-only")
+			proxyOnlyCommand := tests.NewVirtctlCommand("vnc", "--proxy-only", "--namespace", vmi.Namespace, vmi.Name)
+
+			r, w, _ := os.Pipe()
+			os.Stdout = w
```

Comment:
After the json is read from the pipe, the `os.Stdout` should be returned to it's original value. The same in case of a test failure:
```go
oldStdout := os.Stdout
os.Stdout = w
defer func() { os.Stdout = oldStdout }()
```

Please also add a short comment explaining why is the redirection needed.

Reply:
Added